pkg/controller/storagecluster/reconcile.go

```diff
@@ -113,23 +114,18 @@ func newCephCluster(sc *ocsv1alpha1.StorageCluster) *rookCephv1.CephCluster {
 				Enabled:        true,
 				RulesNamespace: "openshift-storage",
 			},
-			// TODO: Enable once StorageClassDeviceSet is merged into Rook
-			/*
-				Storage: rook.StorageScopeSpec{
-					StorageClassDeviceSets: newStorageClassDeviceSets(sc.StorageDeviceSet),
-					// XXX: Depending on how rook-ceph is going to use
-					// StorageClassDeviceSets we would be setting other required parameters
-					// for CephCluster.Storage
-				},
-			*/
+			Storage: rook.StorageScopeSpec{
+				StorageClassDeviceSets: newStorageClassDeviceSets(sc.Spec.StorageDeviceSets),
+				// XXX: Depending on how rook-ceph is going to use
+				// StorageClassDeviceSets we would be setting other required parameters
+				// for CephCluster.Storage
+			},
 		},
 	}
 }
 
-// TODO: Enable once StorageClassDeviceSet is merged into Rook
-/*
-func newStorageClassDeviceSets(dss []ocsv1alpha1.StorageDeviceSet) []rookCephv1.StorageClassDeviceSet {
-	var scds []rookCephv1.StorageClassDeviceSet
+func newStorageClassDeviceSets(dss []ocsv1alpha1.StorageDeviceSet) []rook.StorageClassDeviceSet {
```

Comment:
Please rename this variable to `deviceSets`, `dss` is just a bit too obscure for me.